### PR TITLE
Export Span Scale factor to sbt for Jenkins

### LINKED ISSUE #4223: Fix flakey test from QA testbed
```
The code passed to eventually never returned normally. Attempted 14 times over 5.123310162 seconds. Last failure message: Vector("hola", "hello", "bonjour") was not equal to Vector("hola", "hello", "bonjour", "aurevoir").
```

```
tc: BatchingDbWriter should process again when previous processing finished and we're still over batch size (each time with same test case)
```

https://fc-jenkins.dsp-techops.broadinstitute.org/job/cromwell-test-runner/161/testReport/cromwell.core.actor/BatchActorSpec/BatchingDbWriter_should_process_again_when_previous_processing_finished_and_we_re_still_over_batch_size/

If one needs more info, please talk to @ndbolliger

### PR DESCRIPTION
**Pull request by salonishah11**
Closes #4223

### CODE REVIEW

File: src/ci/bin/test.inc.sh [hunk 1]
```diff
@@ -112,7 +112,7 @@ cromwell::private::create_build_variables() {
             CROMWELL_BUILD_MYSQL_PASSWORD=""
             CROMWELL_BUILD_MYSQL_SCHEMA="cromwell_test"
             CROMWELL_BUILD_GENERATE_COVERAGE=false
-            CROMWELL_BUILD_SPAN_SCALE_FACTOR="15"
+            CROMWELL_BUILD_SPAN_SCALE_FACTOR="20"
```
Comment (mcovarr) [inline src/ci/bin/test.inc.sh]: 😦

**Comment by kshakir** [comment]
I'm not 100% convinced that scaling is completely wired. Have you confirmed that this increase is also affecting your test timeouts? The linked error from #4223 says:

> Attempted 13 times over 5.126574189 seconds.

That's *way* too short for something that should have already been scaled 12x.

I'm in the process of [adding some println-debugging](https://github.com/broadinstitute/cromwell/commit/29d2f35662d6a81a4de383ad54df4ee0242611a4) on a similar issue. In my case I suspect one of the many, many timeouts wasn't scaled for an akka `.ask`, but will have to wait until the docker network issues are resolved to find out.

**Comment by salonishah11** [comment]
@kshakir As per my understanding, `eventually` was the one that was timing out. When I tested it last week with several builds, the test did not fail in any of them. And I had observed that when I decreased the span scale factor, it started failing again. Maybe there are other timeouts failing the test as well. I will try and add println-debugging.

**Comment by kshakir** [comment]
I'm still slowly investigating, but it looks like the scale factor environment variable isn't `export`ed, thus isn't available to `sbt`. Been playing around with the println's in a433a7f20a74faf70a1ec851545f0e9ec6836ce4 ([jenkins](https://fc-jenkins.dsp-techops.broadinstitute.org/job/cromwell-test-runner/555/consoleFull)) and 7fc56d3b73ce537b47aaecc6bf9cd0f1c020646f ([jenkins](https://fc-jenkins.dsp-techops.broadinstitute.org/job/cromwell-test-runner/557/consoleFull)).

**Pull request merged**